sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -3068,6 +3148,17 @@ public WriteResult expand(PCollection<T> input) {
             !getAutoSchemaUpdate(),
             "withAutoSchemaUpdate only supported when using storage-api writes.");
       }
+      if (getRowUpdateFn() != null) {
+        if (method != Method.STORAGE_API_AT_LEAST_ONCE) {
```

Comment:
Why not "checkArgument(method == STORAGE_API_AT_LEAST_ONCE)" instead of if and check?

Reply:
done